src/coverage/format.js

```diff
@@ -0,0 +1,145 @@
+const pad = require('pad')
+const { basename } = require('path')
+
+const ALL_FILES_PATH = '*'
+
+const ERROR_THRESHOLD = 50
```

Comment:
Do you think it would be useful to stat (and read if it exists) a .cgr/.coverage-github-reporter/etc file containing overrides for these values? Maybe this is a 2.0 feature? The consts would be defaults if the config doesn't exist in that case.